docs/install.md

```diff
@@ -936,3 +936,155 @@ This procedure can be used to downgrade the active version of SAT.
     the modified CFS configuration. If more than one CFS configuration was
     modified, use the first one.
 
+## Optional: Configuring SAT to run on an external system over the CAN
+
+It is possible to configure SAT so that it can interact with CSM over the CAN. This allows SAT to run from a physical or
+virtual machine which is not a part of the CSM management cluster.
+
+### Limitations
+
+Some SAT subcommands must interact with services which run on the NCNs in the CSM management cluster, including CSM REST
+APIs, Kubernetes, Ceph, and S3. Certain SAT commands, such as `sat bootsys`, may not work as expected.
+
+Instructions for installing SAT on an external system are provided "as-is", and installing SAT on an external system is
+not an officially supported configuration. These instructions are provided with the hope that they can useful for users
+who desire additional flexibility.
```

Comment:
I think this can be shortened a bit:

```suggestion
Installing SAT on an external system is not an officially supported configuration. These instructions are provided
"as-is" with the hope that they can useful for users who desire additional flexibility.
```

Reply:
I like this, it seems clearer and more concise.